Tests/HTTPTransactionTests.swift

```diff
@@ -86,26 +86,26 @@ class HTTPTransactionTests: XCTestCase {
         let limit = urlValueComponents?.queryItems?.first { $0.name == "limit" }
         XCTAssertNotNil(limit, "The limit should be part of the URL")
         XCTAssertEqual(limit?.value, "80", "The limit should be 80")
-        XCTAssertNotNil(paginationQueryDictionary?["limit"], "The limit should be part of the URL")
-        XCTAssertEqual(paginationQueryDictionary?["limit"], "80", "The limit should be 80")
+        XCTAssertNotNil(queryParamsDictionary?["limit"], "The limit should be part of the URL")
```

Comment:
remove private from the enum QueryParam and use queryParamsDictionary?[HyperwalletBankAccountQueryParam.QueryParam.limit.rawValue]

Reply:
fixed